index.js

```diff
@@ -57,10 +60,27 @@ module.exports = function (options) {
 		}
 
 		try {
-			var errors = checker.checkString(file.contents.toString(), file.relative);
+			var fixResults;
+			var errors;
+			var changed = false;
```

Comment:
@sindresorhus - @rniemeyer and I found that we could use `gulp-changed` *after* the `jscs` call when using the `compareSha1Digest` against the `src` directory (since an auto-formatter like this will most likely be outputting back to the `src` directory vs a traditional gulp `dest` directory). However, this seems more inefficient than simply exposing the `fixed` flag on the `file.jscs` options so something like `gulp-filter` could filter out files that weren't modified by the `jscs` code that ran on them. 

I have no problem with someone using `gulp-changed` upstream of this to limit the input to only source files that have changed, but this seems to be the most appropriate place to set meta-data that reflects what this plugin actually did to the file during its execution (Similar to the `success` flag). Our variable name of `changed` could be renamed to signify the intent better.